recipe/meta.yaml

```diff
@@ -21,26 +20,56 @@ requirements:
     - {{ compiler("c") }}
     - {{ compiler("fortran") }}
     - perl
-    - make                       # [not win]
-    - cmake                      # [win]
-    - curl                       # [win]
-    - jom                        # [win]
+    - make  # [not win]
+    - cmake  # [win]
+    - curl  # [win]
+    - jom  # [win]
   host:
   run:
 
-test:
-  commands:
-    - test -f ${PREFIX}/lib/libopenblas.a                            # [unix]
-    - test -f ${PREFIX}/lib/libopenblas${SHLIB_EXT}                  # [unix]
-    - python -c "import ctypes; ctypes.cdll['${PREFIX}/lib/libopenblas${SHLIB_EXT}']"  # [unix]
-    - if not exist %PREFIX%\\Library\\lib\\openblas.lib exit 1       # [win]
-    - if not exist %PREFIX%\\Library\\bin\\openblas.dll exit 1       # [win]
+outputs:
+  - name: libopenblas
+    build:
+      skip: true  # [win and vc<14]
+      run_exports:
+        - {{ pin_subpackage("libopenblas") }}
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("fortran") }}
+    files:
+      - lib/*{{ SHLIB_EXT }}*  # [not win]
```

Comment:
We don't want `lib/libopenblas${SHLIB_EXT}` in `libopenblas`

Reply:
I probably misunderstood then. I though that all `.so`/`.dylib` should be in `libopenblas`. Why not?